src/GovUk.Education.ExploreEducationStatistics.Admin/Services/FileUploadsValidatorService.cs

```diff
@@ -195,13 +114,95 @@ private bool IsFileExisting(Guid releaseId, ReleaseFileTypes type, string name)
                 .Any(rf => String.Equals(rf.ReleaseFileReference.Filename, name, CurrentCultureIgnoreCase)
                 && rf.ReleaseFileReference.ReleaseFileType == type);
         }
+
+        private async Task<Either<ActionResult, bool>> ValidateDataFileNames(Guid releaseId, string dataFileName, string metaFileName)
```

Comment:
If the Either's right is a `Unit` you don't need to return a `true`.

Reply:
dlone